### ISSUE TITLE
Feature Request: Add Ligature Support

### ISSUE THREAD

**Issue by diantongren** [rektdeckard/departure-mono#12]
Would it be possible to add ligature support to this font? I think it would make the text look more polished and improve readability.

**Comment by minoraxis** [comment]
Hi @diantongren and @alxzh, can you clarify what kinds of ligatures you're looking for? Programming ligatures?

**Comment by diantongren** [comment]
Yes, I'm referring to programming ligatures, which are special character combinations in programming fonts.

**Comment by danicaj3w** [comment]
Hi, I was also looking for programming ligatures and made modifications to include some until official ones were released! You can find them in the ligatures folder here: [Fork with Ligatures](https://github.com/danicaj3w/departure-mono).

**Comment by minoraxis** [comment]
Hi Danica, thanks for taking a pass at this! If/when we get to programming ligatures, we'll be doing it in-house, but appreciate you kicking it off.

If you're open to some feedback/want to tinker further in the meantime, @rektdeckard and I put together some notes. There are a few things I'd address to stay in the spirit of Departure Mono: snapping each character to the pixel grid (7x14), maintaining the fixed pitch (monospace), and reviewing some inconsistency/legibility questions:

<img width="832" alt="Screenshot 2024-12-27 at 1 27 46 PM" src="https://github.com/user-attachments/assets/196bdbff-0c25-4b37-9605-8e7a74243a4f" />

We'll follow up here with updates on official support!
